xyz.armcord.ArmCord.yml

```diff
@@ -0,0 +1,59 @@
+app-id: xyz.armcord.ArmCord
+
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+
+base: org.electronjs.Electron2.BaseApp
+base-version: '22.08'
+
+command: startarmcord
+separate-locales: false
+
+finish-args:
+  - --socket=pulseaudio
+  - --socket=wayland # End users can actually *use* the Wayland backend by setting the USE_WAYLAND environment variable to 1
+  - --socket=x11
```

Comment:
I find this backwards. If it doesn't work with Wayland by default, there's no point in allowing Wayland socket.

Reply:
Does it hurt to have the permission, though? Having it preset is one less thing for the user to do.

Reply:
Yes, I find it misleading. There's no Wayland support if there's a manual action required from the user.

Reply:
The script I provide to start it notifies the user of when Wayland is and isn't being used, wouldn't that be enough? If it's still a worry I'm not against removing the permission, I'll just talk about it a bit in the README.

Reply:
I'm of the opinion that Flathub primarily targets non-technical users, and so anything that doesn't work the default, doesn't deserve to be accounted for.

Reply:
USE_WAYLAND was added upstream, and they force Wayland, so I'll just cut out the check here. Should I keep both permissions, so that their USE_WAYLAND var works OOTB?

Reply:
USE_WAYLAND was added upstream, and they force Wayland, so I'll just cut out the check here. Should I keep both permissions, so that their USE_WAYLAND var works OOTB?